crates/fontawesome/README.md

```diff
@@ -0,0 +1,48 @@
+# fontawesome-free-pack
+
+A redistribution of SVG assets and some metadata from the
+[`@fortawesome/fontawesome-free` npm package](https://www.npmjs.com/package/@fortawesome/fontawesome-free).
+
+## Optimized SVG data
+
+The SVG data is embedded as strings after it is optimized with SVGO. This
+package is intended to easily inject SVG data into HTML documents. Thus, we have
+stripped any `width` and `height` fields from the `<svg>` element, while
+retaining any `viewBox` field in the `<svg>` element.
+
+## Usage
+
+All icons are instantiated as constants using the `Icon` data structure.
+There is a convenient `get_icon()` function to fetch an icon using it's slug name.
+
+Note the family of icon is expected to prefix the slug like a relative path:
+`<family>/<slug>`
+
+### In Python
+
+```python
+from fontawesome_free_pack import get_icon, BRANDS_GITHUB
+
+assert BRANDS_GITHUB.svg == get_icon("brands/github").svg
+```
```

Comment:
Add type safety to these examples. It wouldn't hurt to have some unit tests in python too.